.evergreen/config.yml

```diff
@@ -339,6 +339,101 @@ functions:
           export MONGO_ATLAS_FREE_TIER_REPL_URI_SRV='${MONGO_ATLAS_FREE_TIER_REPL_URI_SRV}'
           ASYNC_RUNTIME=${ASYNC_RUNTIME} .evergreen/run-atlas-tests.sh
 
+  "run ocsp test":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+
+          ASYNC_RUNTIME=${ASYNC_RUNTIME} \
+          MONGODB_URI=${MONGODB_URI} \
+          OCSP_TLS_SHOULD_SUCCEED=${OCSP_TLS_SHOULD_SUCCEED} \
+          OCSP_ALGORITHM=${OCSP_ALGORITHM} \
+          .evergreen/run-ocsp-test.sh
+
+  run-valid-ocsp-server:
+    - command: shell.exec
+      params:
+        script: |
+          cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+          ${PYTHON} -m virtualenv ./venv
+          ./venv/${VENV_BIN_DIR}/python -m pip install -r mock-ocsp-responder-requirements.txt
+    - command: shell.exec
+      params:
+        background: true
+        script: |
+          cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+
+          ./venv/${VENV_BIN_DIR}/python ocsp_mock.py \
```

Comment:
I assume these tasks are mostly copied from another driver's config?

Reply:
They are basically copies of the go driver's tasks, though with some very minor modifications to how python is invoked and whether there is a default value for `$VENV_BIN_DIR`.